src/plugins/clear_button/plugin.scss

```
@@ -7,7 +7,7 @@
         width: 25px;
         top: 0;
         right: calc(#{$select-padding-x} - #{$select-padding-item-x});
-        color: rgba(0, 0, 0);
+        color: rgb(0, 0, 0);
```

This would work better with Bootstrap 5.3.1's new dark mode support, but I'm not sure if you want to incorporate Bootstrap semantics here

```suggestion
        color: var(--bs-emphasis-color, black);
```